.azure-pipelines.yml

```diff
@@ -37,15 +37,13 @@ jobs:
         displayName: Cache pip packages
 
       - script: |
-          python -m pip install --upgrade pip
-          pip install pytest-cov wheel
+          python -m pip install --upgrade pip wheel
```

Comment:
Why is this changing? I think we intentionally did this to make sure we used an upgraded pip to install everything.

Reply:
still the case, this only removes `pytest-cov`